src/scene/graphics/shared/path/GraphicsPath.ts

```diff
@@ -492,8 +497,7 @@ export class GraphicsPath
                 break;
             case 'addPath':
                 // TODO prolly should transform the last point of the path
-                out.x = lastInstruction.data[0].lastX;
-                out.y = lastInstruction.data[2].lastY;
+                lastInstruction.data[0].lastPoint.copyTo(out);
```

Comment:
I believe this is a bug. I discovered it when running the new visual tests after my changes. Looking at the svg visual test which failed, the `GraphicsContext:path` method calls `addPath` of the `_activePath`, passing the path plus a matrix, so the old code referencing `lastInstruction.data[2].lastY` was wrong, it may have just been a completely wrong index to access for the `lastY` as in this case there were only 2 items, the graphics path (which we now look into to get its last point) and a matrix.

Reply:
good catch!